packages/sdk-workflow/src/typing.d.ts

```diff
@@ -49,6 +49,24 @@ declare module "caido:workflow" {
    */
   export type Decision = boolean;
 
+  /**
+   * The input for the JavaScript V2+ Nodes.
+   * @category Data
+   */
+  export type NodeInput = {
+    data?: Bytes;
+    extra?: Record<string, any>;
+  };
```

Comment:
I think we need NodeInputHttp too.
Also please re-export them in the globals, we are not doing imports in the example so that means that any type used must be present globally (check the other file in that directory)